### ISSUE TITLE
Multiple `to` addresses

### ISSUE THREAD

**Issue by HCLarsen** [arcage/crystal-email#72]
Unless I'm misreading the documentation, it only allows for a single email address in `#to`. There are times that people want to send an email to multiple people, without using the CC fields. Might be a useful addition.

**Comment by arcage** [comment]
Yes, `#to`, `#from`, `#cc` or `#bcc` are accept only one address at once.
But, you can call them multiple times.

Thus, you can assign multiple recipients to "To:" header as follows:

```ctystal
email.to "rcpt1@excample.com"
email.to "rcpt2@excample.com"
...
```

**Comment by HCLarsen** [comment]
Thank you for letting me know. Being able to call each multiple times will achieve the functionality.

I do think though, that overloading `#to`, `#cc`, and `#bcc` with versions that take an array of strings would be simpler for users than calling the methods multiple times.

**Comment by arcage** [comment]
I also think the methods that can set multiple recipients at once is useful.

However, I'm worry about the overloads you mentioned above might be a cause of some unexpected behavior because those methods can take optional 2nd `String` type argument for the mailbox name.

```crystal
# When someone wants to set 2 recipients to "To:" header with overloaded method

# correct code
addr_list = ["rcpt1@example.com", "rcpt2@example.com"]
email.to addr_list

# other correct code
email.to ["rcpt1@example.com", "rcpt2@example.com"]

# wrong code(but no error occurs)
email.to "rcpt1@example.com", "rcpt2@example.com"
```

If I do so, I want to adopt deferent method name(eg: `#to_***` or `#***_to`), but don't hit upon a suitable one.

Do you have any ideas for the method name?

**Comment by HCLarsen** [comment]
The thing about that wrong code is that it's a mistake that could already happen, even without an overload that takes an array.

**Comment by arcage** [comment]
Hi,

Sorry for late reply.
I have considered your comment.

Therefore, I have released v0.7.0.

It that version, some methods can take an **Array(String)** or **Array(EMail::Address)** object to set multiple email address at once.

Thank you for your contribution.

**Comment by HCLarsen** [comment]
Great. I look forward to using it.
